docs/myrocks/variables.md

```diff
@@ -370,6 +396,26 @@ Default value is `536870912`.
 
 Maximum value is `9223372036854775807`.
 
+### `rocksdb_bulk_load_fail_if_not_bottommost_level`
+
+| Option       | Description                |
+|--------------|----------------------------|
+| Command-line | --rocksdb_bulk_load_fail_if_not_bottommost_level |
+| Dynamic      | Yes                         |
+| Scope        | Global, Session             |
+| Data type    | Boolean                     |
+| Default      | OFF                         |
+
+The variable was implemented in [Percona Server for MySQL 8.0.33-25](../release-notes/Percona-Server-8.0.33-25.md).
+
+When this variable is enabled, the bulk load fails if the sst files cannot be placed to the bottommost level in the rocksdb. 
+
+This variable can be enabled or disabled only when the [`rocksdb_bulk_load`](#rocksdbbulkload) is `OFF`.
+
+This variable is disabled (OFF) by default.
+
+After you enable `rocksdb_bulk_load_fail_if_not_bottommost_level` variable, disabling this variable may cause severe performance impact.
```

Comment:
`When rocksdb_bulk_load_fail_if_not_bottommost_level is disabled it may cause severe performance impact.`

Reply:
Updated